src/frontend/src/components/EditableTextInputList.tsx

```diff
@@ -9,6 +9,7 @@ import Button from '@mui/material/Button';
 import TextField from '@mui/material/TextField';
 import DeleteIcon from '@mui/icons-material/Delete';
 import IconButton from '@mui/material/IconButton';
+import { Typography } from '@mui/material';
```

Comment:
why switch import syntax? if not intentional, switch to 

`import Typography from '@mui/material/Typography';`

Reply:
see this MUI documentation for reasoning: https://mui.com/material-ui/guides/minimizing-bundle-size/

Reply:
gotcha -- 

1. love the point about DRY. ill go refactor where i can
2. i did some experimenting and wrapping other components in typography then setting some attributes of the typography does change the text
3. i have it auto import typography so thats why theres some inconsistency probably. i'll change to be consistent where i can -- i think i am going to make them all like like 12 here though, because most of the files follow this format.

Reply:
*just read over the documentation for bundle sizing, and it makes sense to actually have imports more specific - so i will move forward like that

Reply:
<img width="284" alt="Screen Shot 2022-12-27 at 1 57 55 AM" src="https://user-images.githubusercontent.com/87815804/209649152-de2ea939-0676-4ba8-aac7-bebb4c479117.png">

a button's text can also be customized using typography. should i keep a typography tag within that or omit it? we certainly like the way the buttons look without it, but should i include for future changes? or i would assume if we want to make changes to a set of buttons in the future we would make some abstraction for a styled button, so maybe i shouldn't include?